src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/DeferredContent/QuickInfoDisplayDeferredContent.cs

```diff
@@ -13,6 +13,7 @@ internal class QuickInfoDisplayDeferredContent : IDeferredQuickInfoContent
         private readonly IDeferredQuickInfoContent _typeParameterMap;
         private readonly IDeferredQuickInfoContent _anonymousTypes;
         private readonly IDeferredQuickInfoContent _usageText;
+        private readonly IDeferredQuickInfoContent _exceptionText;
```

Comment:
Not directly related to this change, but does anyone know why these are typed as `IDeferredQuickInfoContent`, but we always cast them to `ClassifiableQuickInfoContent`?